cypress/e2e/ui/settings.cy.js

```diff
@@ -1,6 +1,48 @@
 /* eslint-disable no-undef */
-describe('GTL', () => {
+describe('Settings > My Settings', () => {
   beforeEach(() => {
     cy.login();
+    cy.menu('Settings', 'My Settings');
+  });
+
+  it('Saves the start page setting', () => {
+    cy.get('#display\\.startpage').then((value) => {
+      expect(value[0].value).to.be.oneOf(['', 'Overview / Dashboard']);
+    });
+    cy.get('#display\\.startpage').click();
+    cy.get('.bx--list-box__menu-item__option').then((options) => {
+      const nums = [...Array(options.length).keys()];
+      nums.forEach((index) => {
+        if (options[index].innerText === 'Overview / Utilization') {
+          cy.get(options[index]).click();
+        }
+      });
+    });
+    cy.get('.bx--form > .bx--btn-set > .bx--btn--primary').click();
+    cy.wait(2000);
+
+    cy.get('#menu_item_logout').click();
+    cy.login();
+    cy.url().should('include', '/utilization');
+
+    cy.menu('Settings', 'My Settings');
+    cy.get('#display\\.startpage').then((value) => {
+      expect(value[0].value).to.equal('Overview / Utilization');
+    });
+    cy.get('#display\\.startpage').click();
+    cy.get('.bx--list-box__menu-item__option').then((options) => {
+      const nums = [...Array(options.length).keys()];
+      nums.forEach((index) => {
+        if (options[index].innerText === 'Overview / Dashboard') {
+          cy.get(options[index]).click();
+        }
+      });
+    });
+    cy.get('.bx--form > .bx--btn-set > .bx--btn--primary').click();
+    cy.wait(5000);
```

Comment:
Same as above, we should figure out what we need to wait for and assert it's ready to continue instead of using a set amount of seconds.